src/Microsoft.AspNetCore.Mvc.Core/ControllerBase.cs

```diff
@@ -1306,15 +1306,15 @@ public virtual ChallengeResult Challenge(AuthenticationProperties properties)
             => new ChallengeResult(authenticationSchemes, properties);
 
         /// <summary>
-        /// Creates a <see cref="ForbidResult"/>.
+        /// Creates a <see cref="ForbidResult"/> (HTTP 403).
```

Comment:
By this logic we wouldn't include any status codes in these comments because any of them could be modified by a middleware. I think we're ok with these being overriden in some cases as long as the convey the intended behavior. @Eilon do you agree?